demos/code/code.js

```diff
@@ -304,45 +306,63 @@ Code.renderContent = function() {
     xmlTextarea.value = xmlText;
     xmlTextarea.focus();
   } else if (content.id == 'content_javascript') {
-    var code = Blockly.JavaScript.workspaceToCode(Code.workspace);
-    content.textContent = code;
-    if (typeof PR.prettyPrintOne == 'function') {
-      code = content.textContent;
-      code = PR.prettyPrintOne(code, 'js');
-      content.innerHTML = code;
-    }
+    try{
+      var code = Blockly.JavaScript.workspaceToCode(Code.workspace);
+      content.textContent = code;
+      if (typeof PR.prettyPrintOne == 'function') {
+        code = content.textContent;
+        code = PR.prettyPrintOne(code, 'js');
+        content.innerHTML = code;
+      }
+    }catch(err){
+      alert(err);}
```

Comment:
Use Blockly.alert(..) so there is a clear override point for apps that use this demo with their own UI or in ways that cannot allow blocking the execution environment.